builder/main.py

```
@@ -121,17 +158,25 @@ def _parse_size(value):
         return int(value[:-1]) * base
     return value
 
+
 def _parse_partitions(env):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)